pykern/pkresource.py

```diff
@@ -8,34 +8,97 @@
 
 # Root module: Import only builtin packages so avoid dependency issues
 import errno
-import inspect
+import glob
+import importlib
 import os.path
 import pkg_resources
-import re
 
+# TODO(e-carlin): discuss with rn if ok to import pkio
 from pykern import pkinspect
+from pykern import pkio
 from pykern import pksetup
 
 
-def filename(relative_filename, caller_context=None):
+def file_path(relative_filename, caller_context=None, packages=None):
```

Comment:
This will break things. Leave old routine until everything is fixed. I was just using filename() in rsdockerspawner, and don't want to coordinate.